types/src/v0/traits.rs

```diff
@@ -493,6 +496,9 @@ pub trait SequencerPersistence: Sized + Send + Sync + Clone + 'static {
                 ViewNumber::genesis()
             }
         };
+
+        // TODO load from storage
+        let next_epoch_high_qc = None;
```

Comment:
@imabdulbasit here I think we need to add something to our storage layer to store this type of QC.

Reply:
yeah, I can add it

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/2422